src/PSMDescription.cpp

```diff
@@ -22,12 +22,12 @@
 #include "DescriptionOfCorrect.h"
 
 PSMDescription::PSMDescription() :
-    features(NULL), expMass(0.), calcMass(0.), scan(0),
+    features(NULL), expMass(0.), calcMass(0.), retentionTime_(nan("")), scan(0),
```

Comment:
Why are we initializing to a nan?

Reply:
I think Lucas implemented it. I guess we want to use nan if there is no rt in the input file.